app/controllers/application.js

```diff
@@ -1,6 +1,15 @@
 import Ember from 'ember';
 import OSFAgnosticAuthControllerMixin from 'ember-osf/mixins/osf-agnostic-auth-controller';
 
+/**
+ * @module ember-preprints
+ * @submodule controllers
+ */
+
+/**
+ * @class Application Controller
+ * @extends Ember-OSF.OSFAgnosticAuthControllerMixin
```

Comment:
Technically, this extends Ember.Controller. For mixin, a better option might be `@uses`.

Here is a usage example from ember showing extends vs uses in action:
https://github.com/emberjs/ember.js/blob/v2.11.0/packages/ember-runtime/lib/system/array_proxy.js#L74